checkov/main.py

```diff
@@ -36,12 +37,17 @@
 logger = logging.getLogger(__name__)
 checkov_runner_module_names = ['cfn', 'tf', 'k8', 'sls', 'arm', 'tf_plan', 'helm']
 checkov_runners = ['cloudformation', 'terraform', 'kubernetes', 'serverless', 'arm', 'terraform_plan', 'helm',
-                   'dockerfile']
+                   'dockerfile', 'secrets']
 
 # Check runners for necessary system dependencies.
 runnerDependencyHandler = RunnerDependencyHandler(checkov_runner_module_names, globals())
 runnerDependencyHandler.validate_runner_deps()
 
+USE_SECRETS_RUNNER = os.environ.get("CHECKOV_USE_DETECT_SECRETS", "FALSE")
```

Comment:
Two comments:

1. Why not use a regular config option for this? (so they can use a CLI flag or the config file)?
2. Are we sure we want to have a different way to enable this runner? All the other runners are enabled / disabled using --framework or --skip-framework